activerecord/test/cases/scoping/default_scoping_test.rb

```diff
@@ -422,6 +422,11 @@ def test_unscoped_with_joins_should_not_have_default_scope
                  Comment.joins(:post).to_a
   end
 
+  def test_unscoping_an_array_default_column
```

Comment:
I think this is better put in `relations_test`, you also don't need a subclass or scope:

```ruby
def test_unscope_grouped_where
  posts = Post.where(
    title: ["Welcome to the weblog", "So I was thinking", nil]
  )

  assert_equal 2, posts.count
  assert_equal Post.count, posts.unscope(where: :title).count
end
```